templates/predict.ipynb

```diff
@@ -21,34 +21,27 @@
    ],
```

Comment:
<p>How long does this take to make predictions for a single 2048x2048x1 image? 2048x2048x15?</p><p> </p><p>What about on a CPU workstation? i.e. should we tell people they definitely need a GPU?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/cell_classification/pull/65/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
RTX 3090 24GB on a workstation: 54 seconds for 1 FoV (2048x2048x35), so ~1.5sec for 1 image with test time augmentations (TTA)
RTX3050 4GB on a notebook: 150seconds for one FoV or ~4.3sec per image with TTA
i7 11800H 8-core CPU on notebook: 44 minutes for one FoV or 75sec per image with TTA. Without TTA, 3.3 sec per image 115 sec per FoV
I'll add text, saying that CPU inference is okay without TTA, but GPU inference is always preferred.

Reply:
<p>TTA makes it 25x slower? 3.3 sec -&gt; 75 seconds?</p><p> </p><p>Yeah I agree, and it's great that it'll work decently on CPU</p>

Reply:
That's a bit extreme, the difference should be 6-fold, because TTA averages over 6 different views of the data. I guess the 25-fold increase is due to CPU throttle, my laptop got pretty hot from running these experiments.